easybuild/easyconfigs/s/SeqAn/SeqAn-2.4.0-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,50 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+# Updated: Jakub Zarybnicky (INUITS), Denis Kristak (INUITS)
+easyblock = 'CMakeMake'
+
+name = 'SeqAn'
+version = '2.4.0'
+
+homepage = 'https://github.com/seqan/seqan'
+
+description = """
+ SeqAn is an open source C++ library of efficient algorithms and data structures
+ for the analysis of sequences with the focus on biological data.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = ['https://github.com/seqan/seqan/archive/']
+sources = ['%(namelower)s-v%(version)s.tar.gz']
+checksums = ['d7084d17729214003e84818e0280a16f223c8f1c6a30eeef040c27e0c0047bd7']
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('CMake', '3.21.1'),
+    ('Python', '3.9.6'),
+]
+
+dependencies = [
+    ('bzip2', '1.0.8'),
+    ('zlib', '1.2.11'),
+]
+
+separate_build_dir = True
+
+postinstallcmds = [
+    "mkdir -p %(installdir)s/util/cmake",
+    "cp -a %(builddir)s/seqan-*/util/cmake/seqan-config.cmake %(installdir)s/util/cmake/",
+    "cp -a %(builddir)s/seqan-*/include %(installdir)s",
+]
+
+sanity_check_paths = {
+    'files': ["bin/%s" % x for x in ["gustaf_mate_joining", "mason_materializer", "mason_variator",
```

Comment:
Can we add a sanity check command like `--help` for any of these?

Reply:
added